CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Comment line immediately above ownership line is reserved for related gus information. Please be careful while editing.
+#ECCN:Open Source
```

Comment:
Is this meant to be added to https://github.com/SalesforceCommerceCloud/raml-toolkit/blob/main/.github/CODEOWNERS ?